submissions/sadoharu/task_js_dom_/scripts.js

```diff
@@ -0,0 +1,73 @@
+window.addEventListener("DOMContentLoaded", () => {
+    const pages = document.querySelectorAll(".section__page");
+    const navList = document.querySelector(".nav-bar__list");
+  
+    const navListItem = [
+      {
+        title: "Overview",
+        icon: "icon-browser",
+      },
+      {
+        title: "About",
+        icon: "icon-about-dot-me",
+      },
+      {
+        title: "Achievement",
+        icon: "icon-award",
+      },
+      {
+        title: "Projects",
+        icon: "icon-files-empty",
+      },
+      {
+        title: "Reviews",
+        icon: "icon-rate_review",
+      },
+    ];
+  
+    navListItem.forEach((item, i) => {
+      const liItem = document.createElement("li");
+      liItem.classList.add("nav-bar__list_item");
+      liItem.tabIndex = 0;
+      liItem.setAttribute("data-index", i);
+      liItem.innerHTML = `
+              <span class="nav-bar__list_icon ${item.icon}"></span>
+              <span class="nav-bar__list_title">${item.title}</span>
+              `;
+      navList.append(liItem);
+    });
+  
+    const togglePageVisibility = (show = 0) => {
+      pages.forEach((item) => {
+        item.classList.remove("visible");
+      });
+      pages[show].classList.add("visible");
+    };
+  
+    togglePageVisibility();
+  
+    const togglePage = (e, node) => {
+      togglePageVisibility(node.getAttribute("data-index"));
+      e.currentTarget
+        .querySelectorAll(".nav-bar__list_item")
+        .forEach((item) => item.classList.remove("active"));
+      node.classList.add("active");
+    };
+  
+    navList.addEventListener("click", (e) => {
+      if (e.target.getAttribute("data-index")) {
+        togglePage(e, e.target);
+      } else if (
+        e.target.closest(".nav-bar__list_item").getAttribute("data-index")
+      ) {
+        togglePage(e, e.target.closest(".nav-bar__list_item"));
+      }
+    });
+  
+    document.addEventListener("keyup", (e) => {
+      if (e.code === "Space") {
+        togglePage(e, e.target);
+      }
+    });
```

Comment:
`Space` on `document`? Please no :) It's way too much. Space used for scrolling pages, as an example. No need in changing existing patterns without reasons.

Reply:
I can remove this line of code by adding buttons

Reply:
And how buttons will help you?

Reply:
I can select any item from the menu using only the 'tab' and 'space' keys, without mouse

Reply:
Considering the fact, that it's a menu, it should be something like this:

```
nav
  ul
    li
      a
    li
      a
    li
      a
  ```
  
  In such a case with tab you will go thru anchors

Reply:
Instead of the a tag, I used button